judge/migrations/0109_contest_decimal_points.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 2.2.7 on 2020-03-30 01:44
+
+import django.core.validators
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def int_to_float(apps, schema_editor):
+    ContestParticipation = apps.get_model('judge', 'ContestParticipation')
+    for participation in ContestParticipation.objects.all():
+        participation.recompute_results()
```

Comment:
Object methods don't exist in a migration - You'll need to copy the body of `recompute_results` over.